web/crux-ui/src/components/products/edit-product-card.tsx

```diff
@@ -45,10 +45,12 @@ const EditProductCard = (props: EditProductCardProps) => {
   const handleApiError = defaultApiErrorHandler(t)
 
   const formik = useFormik({
-    validationSchema: !editing ? createProductSchema : updateProductSchema,
     initialValues: {
       ...product,
     },
+    validationSchema: !editing ? createProductSchema : updateProductSchema,
+    validateOnChange: false,
```

Comment:
would be nicer to make a custom hook like `useDyoFormik` and set these booleans to false as a default